### PR TITLE
Mention origin and extent tutorial in API docs for origin kwarg

### PR DESCRIPTION
**Pull request by ngoldbaum**
## PR Summary

This adds a link to the origin and extent tutorial. There's already a link for the extent kwarg below this but I don't think it hurts to have two links since at least for myself I tend to go tunnel vision in these API docs pages and ignore kwargs I don't think I need to care about.

## PR Checklist

- [x] Documentation is sphinx and numpydoc compliant

### CODE REVIEW

**Comment by jklymak** [comment]
Looks good but one of your lines is too long.

**Comment by ngoldbaum** [comment]
Oops, fixed.

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -5519,6 +5519,9 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             Note that the vertical axes points upward for 'lower'
             but downward for 'upper'.
 
+            See the example :doc:`/tutorials/intermediate/imshow_extent` for
+            examples and a more detailed description.
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py]:
```suggestion
            a more detailed description.
```

I would keep this exactly the same as in the sentence in the `extent`parameter description. Twice using example is kind of redundant. :smile: 

https://21987-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.axes.Axes.imshow.html#matplotlib.axes.Axes.imshow

Comment (ngoldbaum) [reply lib/matplotlib/axes/_axes.py]: I'll keep them consistent but I think the current spelling is actually a little confusing.

**Pull request merged**